ZenPacks/zenoss/Microsoft/Windows/modeler/plugins/zenoss/winrm/WinMSSQL.py

```diff
@@ -293,88 +294,56 @@ def __init__(self, conn_info, log):
         "  $ag_info['id'] = $ag_uid;"
         "  $ag_info['ag_res_id'] = $ag_res_uid;"
         "  $ag_info['primary_replica_server_name'] = $pr_rep_servr_name;"
-        # Put databases on top beacause we get them regardless of Replica role (either on Primary or Secondary Replicas)
```

Comment:
Database related script block I moved on the bottom of the script as now we collect info from all Windows nodes on which Availability Group is placed.